arch/arm/configs/nati_bluefin_defconfig

```diff
@@ -60,7 +60,6 @@ CONFIG_BRIDGE_EBT_MARK_T=y
 CONFIG_BRIDGE=y
 CONFIG_BRIDGE_VLAN_FILTERING=y
 CONFIG_NET_DSA=y
-CONFIG_NET_DSA_MV88E6XXX_ONLY_8021AS=y
```

Comment:
I assume this got dropped because of `savedefconfig`? Does that mean you don't require the https://github.com/ni/linux/commit/295c911ba097730c1f3fc0974145b50bc5e414bc cherrypick? I was wondering if you needed that when doing 6.12 [rebase](https://github.com/ni/linux/pull/211).

Reply:
Yes, these are the results of `savedefconfig`. My understanding is that the 6.6 branch already has upstream DSA changes which obviates the need for https://github.com/ni/linux/commit/295c911ba097730c1f3fc0974145b50bc5e414bc. It is very likely a mistake that it was brought over from 4.1. Should I revert the commit?

Reply:
Ok, I think this is fine.